projects/ngx-echarts/src/lib/ngx-echarts.directive.ts

```diff
@@ -17,9 +17,13 @@ import ResizeObserver from 'resize-observer-polyfill';
 import { Observable } from 'rxjs';
 import { switchMap } from 'rxjs/operators';
 import { ChangeFilter } from './change-filter';
+import * as echarts from 'echarts';
```

Comment:
I'm afraid it will break custom build

Reply:
Why? I had no problems. Compiled via the "npm run build: lib" command.

Reply:
It will dramatically increase the package size of the project which installed `ngx-echarts` but only import some echarts by demand. 
You can refer to the ECharts official document for details: https://echarts.apache.org/en/tutorial.html#Create%20Custom%20Build%20of%20ECharts

Reply:
It seems that I know how to solve these problems. To reduce the package size, you can use import type.
`import type * as somePackage from 'some-package'`
[typescript documentation](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html)